js/index.js

```diff
@@ -15,9 +20,24 @@ function calculateAll() {
 
   // ITERATION 2
   //... your code goes here
+  let products=document.querySelectorAll(".product")
+  products.forEach((product)=>{
+    updateSubtotal(product)
+  })
 
   // ITERATION 3
   //... your code goes here
+  let allSubtotals=document.querySelectorAll(".subtotal span")
+  console.log(allSubtotals)
+
+  let totalPrice=0
+
+  allSubtotals.forEach((subtotal)=>{
```

Comment:
If you would return the subtotal value from the updateSubtotal-function, you don't need this second loop